### PR TITLE
smirnoff99Frosst-1.1.0

### PR DESCRIPTION
**Pull request by j-wags**
## Summary
Adds `smirnoff99Frosst-1.1.0.offxml` per https://github.com/openforcefield/openforcefield/pull/368 (copied from `smirnoff99Frosst-experimental.offxml` from 0.4.1 OFFTK release). 

This file includes several changes to hierarchy and other aspects of FF. 

Spec upgrade was performed programmatically, beginning with `smirnoff99Frosst-1.0.9.offxml`, then reading it using the OFFTK version 0.4.0. Contents were then modified by hand by @davidlmobley 

@j-wags Hand-copied in-text comments from 1.0.9 to 1.1.0

@j-wags Added hydrogen bond length constraints by hand.

## Status
- [x] New file is readable by toolkit
- [x] Validate that hydrogen bond length constraints operate as intended
- [x] Ready for review
- [ ] Ready for merge

### CODE REVIEW

File: smirnoff99frosst/offxml/smirnoff99Frosst-1.1.0.offxml [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+<SMIRNOFF version="0.3" aromaticity_model="OEAroModel_MDL">
+    <Author>C. I. Bayly, OpenEye/UC Irvine; C. C. Bannan, UC Irvine; D. L. Mobley, UC Irvine; J. R. Wagner, Open Force Field/UC Irvine</Author>
+    <Date>Date: July 25, 2019</Date>
+    <Constraints version="0.3">
+      <!-- constrain all bonds to hydrogen to their equilibrium bond length -->
+      <Constraint smirks="[#1:1]-[*:2]" id="c1"/>
+    </Constraints>
+    <!-- This file is meant for processing via openforcefield.typing.engines.smirnoff -->
+    <!-- WARNING: AMBER functional forms drop the factor of 2 in the bond energy term, so cross-comparing this file with a corresponding .frcmod file, it will appear that the values here are twice as large as they should be. -->
+	<Bonds version="0.3" potential="harmonic" fractional_bondorder_method="None" fractional_bondorder_interpolation="linear">
```
Comment (jchodera) [inline smirnoff99frosst/offxml/smirnoff99Frosst-1.1.0.offxml]: This is only cosmetic, but it looks like indentation is not consistent.

Comment (j-wags) [reply smirnoff99frosst/offxml/smirnoff99Frosst-1.1.0.offxml]: Noooo! It's tabs vs. spaces. Fie on thee, `xmltodict`. Fie on your entire house!

File: README.md [hunk 2]
```diff
@@ -105,17 +119,19 @@ Additionally, with ParmEd, it should be possible to convert parameterized OpenMM
...
 - [Version 1.0.9](https://doi.org/10.5281/zenodo.3256444): Addresses [issue 89](https://github.com/openforcefield/smirnoff99Frosst/issues/89): Fixes torsion t56, where SMIRKS `[!1:1]-[#7X4,#7X3:2]-[#6X4;r3:3]-[*:4]` should be `[!#1:1]-[#7X4,#7X3:2]-[#6X4;r3:3]-[*:4]`. The first means "not an isotope with mass 1" (`!1`), but we intend for it to apply to all hydrogens, so it has been changed to `!#1`.
-**Not yet in a version**:
+- Version 1.1.0: Adds hydrogen bond constraints. Updates contents to SMIRNOFF 0.3 spec (spec change performed by OFF toolkit 0.4.0 reading smirnoff99Frosst-1.0.9, and then writing the `ForceField` object out in the 0.3 spec). Fixes several issues with FF hierarchy, SMIRKS patterns. Full details available in [openforcefield/openforcefield Issue 367](https://github.com/openforcefield/openforcefield/issues/367).
```
Comment (davidlmobley) [inline README.md]: I would add after "SMIRKS patterns", the sentence "This fixes some issues with coverage of certain areas of chemical space, and parameters which could never be utilized due to hierarchy issues."

Comment (j-wags) [reply README.md]: Added in 0f29c79

**Pull request merged**